dplace_app/tests/test_renderers.py

```diff
@@ -19,7 +19,7 @@ def test_field_names_for_cultural_variable(self):
             assert fieldnames[k].endswith(expected)
     
     def test_field_names_for_environmental_variable(self):
-        name = self.renderer.field_names_for_environmental_variable(self.var)
+        name = self.renderer.field_names_for_environmental_variable({'variable': self.var})
```

Comment:
Not entirely sure if I should have done this, but it was the only way I could get the tests to pass. Change/fix if should not have been changed.